### ISSUE TITLE
Is there new brand?

### ISSUE THREAD

**Issue by CordlessWool** [DBook/website#1]
is there any new brand now or will you release it to github? I am so sad about it. I planed to write my Mastertheses with it....

**Comment by ribalba** [comment]
I still have to clean some stuff out of the code for the FOSS release. If you know how to set up a simple server you can run it yourself. I can share the code with you.

**Comment by CordlessWool** [comment]
Yes I could. Will run it in a docker container if possible? Would be great if you could share it with me

**Comment by ribalba** [comment]
Send my your gitlab username to didi@ribalba.de and I will add you
